.coderabbit.yaml

```
@@ -0,0 +1,41 @@
+# yaml-language-server: $schema=https://coderabbit.ai/integrations/schema.v2.json
+
+language: ko-KR
+early_access: false
+#tone_instructions:
```

YAMLlint (1.35.1)

[warning] missing starting space in comment

(comments)